HerPublicWebsite.BusinessLogic/Models/ReferralRequest.cs

```diff
@@ -28,6 +28,7 @@ public class ReferralRequest
     public string ContactEmailAddress { get; set; }
     public string ContactTelephone { get; set; }
 
+    public bool PendingLocalAuthorityWhenSubmitted { get; set; }
     public DateTime RequestDate { get; set; }
```

Comment:
For consistency with how you assign them in the constructor, please put this new boolean underneath `RequestDate`.

```
public DateTime RequestDate { get; set; }
public bool PendingLocalAuthorityWhenSubmitted { get; set; }
```